### ISSUE TITLE
Running multiple currency pairs on the same exchange account

### ISSUE THREAD

**Issue by FingerMoney** [bwentzloff/trading-bot#45]
If Im understanding the code correctly, we can only run one bot per exchange account because we cant designate a budget or portion of the balance to use. In order to run multiple bots, say one for XMR and one for ETH, simultaneously we would need additional exchange accounts, correct?

**Comment by r-neri** [comment]
No. you use 2 bots one for every pair of coins. Every bot will use your aceess key for your exchange.
